syz-manager/stats.go

```diff
@@ -4,6 +4,8 @@
 package main
 
 import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
```

Comment:
Non-standard imports are placed after standard library and separated by an empty line.